crates/warg/src/package.rs

```diff
@@ -1,15 +1,13 @@
-use prost::Message;
+use crate::{hash, protobuf, Decode, Encode, Signable};
 use anyhow::Error;
+use prost::Message;
 use thiserror::Error;
 
-use crate::hash;
-
 pub mod model;
 pub mod validate;
 
-use crate::{protobuf, Decode, Encode, Signable};
-
-// Deserialization
+/// The currently supported package protocol version.
+pub const PACKAGE_RECORD_VERSION: u32 = 0;
```

Comment:
If we're going to add a version constant for package records, operator records should get one too.

Reply:
Agreed. I was waiting on touching the operator log code to see if we even wanted to take changes to the package log validation before doing so. I'll have a follow-up for that.